src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -379,18 +380,22 @@ def is_error_count_healthy(ethreshold: EnrichedThreshold, timeseries: List[Dict[
         if parser.parse(i["time"]) > ethreshold["end"]:
             # timeseries are ordered chronologically
             # So if we're past our threshold.end, we can skip the rest
+            logger.info("Reached end of threshold window. Breaking")
```

Comment:
hmm instead of doing this couldn't we get the min between the `query_window["end"]` and `threshold.end` when getting the error counts from `get_errors_counts_timeseries_by_project_and_release`? we could fetch less data that way too? or i may be misunderstanding

Reply:
i see, makes sense. thanks!